readthedocs/doc_builder/director.py

```diff
@@ -654,6 +662,7 @@ def get_rtd_env_vars(self):
             # "READTHEDOCS_GIT_HTML_URL": self.data.project.remote_repository.html_url,
             "READTHEDOCS_GIT_IDENTIFIER": self.data.version.identifier,
             "READTHEDOCS_GIT_COMMIT_HASH": self.data.build["commit"],
+            "READTHEDOCS_PRODUCTION_DOMAIN": settings.PRODUCTION_DOMAIN,
```

Comment:
This kinda feels like an internal thing, especially not documenting it. I assume it's needed for something downstream?

Reply:
This is required by the `sphinx-build-compatibility` extension to know if we have to perform the APIv2 call to community or business. I may miss the documentation for this, but I will add it in that case. Seems useful to pass it to the build commands.